### ISSUE TITLE
Scheduler not working automatically

### ISSUE THREAD

**Issue by devvz** [SpaceCP/Bugs#239]
## Expected Behavior

I have 2 scheduled tasks set up to run that will warn and then restart the server at 3:55 AM and 4:00 AM respectively. Manually testing these tasks work as intended.

## Current Behavior

Scheduled tasks are not automatically running on their scheduled times. The triggers are set up as they should (trigger at specific time).

## Steps to Reproduce (for bugs)

1. Make scheduled task for 55 3 \* \* \* and 0 4 \* \* \* (3:55 AM and 4:00 AM)
2. Wait until time, but server does nothing
## Context

I am trying to restart my server daily, to free up some memory and refresh the server.

## Your Environment

- Daemon version used: Latest
- Server OS: Ubuntu Server 16.04
- Server environment: node.js 5.12
  
- Other info:
## Screenshots

![image](https://cloud.githubusercontent.com/assets/23101714/19772542/94ac7ca2-9c34-11e6-80fd-02eb0bdef053.png)

**Comment by JamyDev** [comment]
It uses the timezone that is set on the server. Are you sure this is the same as your timezone?

**Comment by JamyDev** [comment]
It uses the timezone that is set on the server. Are you sure this is the same as your timezone?

**Comment by JamyDev** [comment]
It uses the timezone that is set on the server. Are you sure this is the same as your timezone?

**Comment by devvz** [comment]
Yes, this is the same as my time zone. Even if it wasn't, the task has never ran automatically.

**Comment by ryan77627** [comment]
I can confirm this, I have two scheduler tasks set up to run at 58 22 and 59 22 everyday but they do not run. I checked the timezone and it is correct, The system time is correct, the hardware time is correct, the crontab setup on the system itself runs fine. The only thing is, the webpanel has the wrong time markings. If i go into the console from the webpanel, it has the right times because it is a direct output from the server itself. But if i go into the file manager on the daemon the files last modified dates are all screwed up.
